iocore/net/UnixNet.cc

```diff
@@ -69,7 +69,20 @@ class InactivityCop : public Continuation
         continue;
       }
 
+      // set a default inactivity timeout if one is not set
+      if (ne->next_inactivity_timeout_at == 0 && nh.config.default_inactivity_timeout > 0) {
+        Debug("inactivity_cop", "vc: %p inactivity timeout not set, setting a default of %d", ne,
+              nh.config.default_inactivity_timeout);
+        ne->set_inactivity_timeout(HRTIME_SECONDS(nh.config.default_inactivity_timeout));
+        ne->default_inactivity_timeout = true;
+        NET_INCREMENT_DYN_STAT(default_inactivity_timeout_applied_stat);
+      }
+
       if (ne->next_inactivity_timeout_at && ne->next_inactivity_timeout_at < now) {
+        if (ne->default_inactivity_timeout) {
```

Comment:
This might not be completely accurate since something could have set the inactivity timeout to something that is not the default before we reach this point.

Reply:
fixed it. thanks Bryan